Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiDisplay.cpp

```diff
@@ -329,48 +329,25 @@ namespace AzToolsFramework::ViewportUi::Internal
     {
         auto viewportUiMapElement = m_viewportUiElements.find(elementId);
 
-        if (viewportUiMapElement != m_viewportUiElements.end() &&
-            viewportUiMapElement->second.m_widget)
+        if (viewportUiMapElement != m_viewportUiElements.end() && viewportUiMapElement->second.m_widget)
         {
             viewportUiMapElement->second.m_anchored = true;
             SetUiOverlayContentsAnchored(viewportUiMapElement->second.m_widget.get(), alignment);
         }
     }
 
-    void ViewportUiDisplay::AddMaximumSizeViewportUiElement(QPointer<QWidget> widget)
-    {
-        if (widget)
-        {
-            return;
-        }
-
-        if (m_fullScreenWidget)
-        {
-            AZ_Warning(
-                "ViewportUi", false,
-                "Attaching a maximum size element when one already exists. Removing the previously attached element.");
-            RemoveViewportUiElement(GetViewportUiElementId(m_fullScreenWidget));
-        }
-
-        widget->setAttribute(Qt::WA_ShowWithoutActivating);
-        widget->setParent(&m_uiOverlay);
-        m_fullScreenWidget = widget;
-        m_fullScreenLayout.addWidget(m_fullScreenWidget, 0, 0, 1, 1);
-        m_renderOverlay->setFocus();
-    }
-
     // disables system background for widget and gives a transparent background
-    static void ConfigureWidgetForViewportUi(QPointer<QWidget> widget)
+    static void ConfigureWindowForViewportUi(QPointer<QMainWindow> mainWindow)
     {
-        // no background for the widget else each set of buttons/textfields/etc would have a black box around them
-        SetTransparentBackground(widget);
-        widget->setWindowFlags(Qt::Window | Qt::FramelessWindowHint);
+        // no background for the widget else each set of buttons/text-fields/etc would have a black box around them
+        SetTransparentBackground(mainWindow);
+        mainWindow->setWindowFlags(Qt::Window | Qt::FramelessWindowHint | Qt::WindowDoesNotAcceptFocus);
```

Comment:
So I don't lose it again: the actual fix is here.

Reply:
Totally! Yes sorry I do usually try to do this I think I just think I wound up poking around so much and wanted to fix all these little bits and bobs I saw..

Maybe I can re-post this as two separate PRs so I follow my own rules! 😝 (I think I was just so elated I fixed it I just wanted to post the fix ASAP 😅)

Reply:
You are forgiven.